src/app/core/footer/footer.component.ts

```diff
@@ -1,15 +1,10 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-footer',
   templateUrl: './footer.component.html',
   styleUrls: [ './footer.component.scss' ],
 })
-export class FooterComponent implements OnInit {
-
+export class FooterComponent {
   constructor() { }
```

Comment:
let remove redundant empty constructors and LC hooks!

Reply:
Will do